instaloader/__main__.py

```diff
@@ -140,6 +145,11 @@ def _main(instaloader: Instaloader, targetlist: List[str],
     if latest_stamps_file is not None:
         latest_stamps = LatestStamps(latest_stamps_file)
         instaloader.context.log(f"Using latest stamps from {latest_stamps_file}.")
+    # load cookies if browser is not None
+    if browser and bc3_library:
+        import_session(browser.lower(), instaloader, cookiefile)
+    elif browser and bc3_library is False:
```

Comment:
```suggestion
    elif browser and not bc3_library:
```

I think `not ...` is a littler more pythonic and cooler than `... is False`.

Reply:
Done.